src/utils.js

```diff
@@ -49,3 +51,106 @@ export function readJsonFile(name) {
   return JSON.parse(readFileSync(name, 'utf8'));
 }
 
+function readCompressedToBuffer(path) {
+  return new Promise((resolve, reject) => {
+    const gzipStream = createReadStream(path).pipe(zlib.createGzip({
+      level: zlib.constants.Z_BEST_COMPRESSION
+    }));
+    const buffers = [];
+
+    gzipStream.on('data', data => {
+      buffers.push(data);
+    });
+    gzipStream.on('error', err => {
+      reject(err);
+    });
+    gzipStream.on('end', () => {
+      resolve(Buffer.concat(buffers));
+    });
+  });
+}
+
+function sendFileToSlack(path, title, filename) {
+  if (!process.env.NPM_CI_SLACK_TOKEN) {
```

Comment:
will this break the build or fail gracefully?

Reply:
This will just print an error but continue, it's wrapper in a `try {} catch` https://github.com/wix/npm-ci-scripts/pull/17/files/64fe5ccabcc4644206024093f1caefe64046d11f#diff-2b4ca49d4bb0a774c4d4c1672d7aa781R127